css/components/download-item.postcss

```diff
@@ -0,0 +1,28 @@
+/*----------------------------------------*\
+  Download Item
+\*----------------------------------------*/
+
+.download-item {
+  @apply flex pl-0 pb-4 pt-4 align-top border-b border-gray-200 text-left w-full;
```

Comment:
I try to regroup logically the properties for better readability. What about
```
@apply flex w-full;
@apply pl-0 pb-4 pt-4;
@apply border-b border-gray-200;
```
`align-top`can be removed

Reply:
check and check